component/prometheus/exporter/kafka/kafka.go

```diff
@@ -0,0 +1,153 @@
+package kafka
+
+import (
+	"fmt"
+	"github.com/Shopify/sarama"
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/component/discovery"
+	"github.com/grafana/agent/component/prometheus/exporter"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/kafka_exporter"
+	config_util "github.com/prometheus/common/config"
+)
+
+var DefaultArguments = Arguments{
+	UseSASLHandshake:        true,
+	KafkaVersion:            sarama.V2_0_0_0.String(),
+	MetadataRefreshInterval: "1m",
+	AllowConcurrent:         true,
+	MaxOffsets:              1000,
+	PruneIntervalSeconds:    30,
+	TopicsFilter:            ".*",
+	GroupFilter:             ".*",
+}
+
+type Arguments struct {
+	Instance string `river:"instance,attr,optional"`
+	// Address array (host:port) of Kafka server
```

Comment:
These comments are not needed IMHO, but you can keep them if you want. Maybe at least start the comment with the name of the field?